runtime/frequency/src/lib.rs

```diff
@@ -607,9 +607,9 @@ impl pallet_democracy::Config for Runtime {
 		frame_system::EnsureRoot<AccountId>,
 	>;
 
-	/// A super-majority of 3/5ths can have the next scheduled referendum be a straight majority-carries vote.
+	/// A simple-majority of 50% + 1 can have the next scheduled referendum be a straight majority-carries vote.
 	type ExternalMajorityOrigin = EitherOfDiverse<
-		pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 5>,
+		pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 1, 2>,
```

Comment:
wouldn't this also work exactly on 50%?

Reply:
Ah it would. I should use `EnsureProportionMoreThan` to make it 50% + 1

Great catch. Fixing.

Reply:
Fixed!